Products/ZenModel/migrate/updateMemcachedMaxConn.py

```diff
@@ -0,0 +1,50 @@
+#############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+log = logging.getLogger("zen.migrate")
+
+import re
+import Migrate
+from Products.ZenModel.ZMigrateVersion import (
+SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+)
```

Comment:
These need to be written as
```
from Products.ZenModel.ZMigrateVersion import SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
```
All on one line, otherwise, there will be errors when the prodbin component is released.

Reply:
This change was made.